rc/tools/git.kak

```diff
@@ -326,3 +337,124 @@ define-command -params 1.. \
             ;;
     esac
 }}
+
+# Options needed by git-diff-goto-source command
+declare-option -hidden str git_diff_hunk_filename
+declare-option -hidden int git_diff_hunk_line_num_start
+declare-option -hidden int git_diff_go_to_line_num
+declare-option -hidden str git_diff_git_dir
+declare-option -hidden str git_diff_section_heading
+declare-option -hidden int git_diff_cursor_column
```

Comment:
Not terribly happy about the amount of hidden options necessary to make this work, but not a big deal either.